### PR TITLE
🔀🌐 [en_GB] add/update translations for multiple modules

### PR DESCRIPTION
**Pull request by scottyboii**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* en_GB

**What is included in your update?**
* New documentation for en_GB: 
  * extendedMap
  * generalExtensions
  * heatmap
  * hotkeys
  * messageTemplates
* Updated en_GB documentation for extendedCallList

**Additional notes**
Wrote these up back in may but forgot to submit them, so there may be some information missing but figured they're better to be used.

### CODE REVIEW

File: src/modules/messageTemplates/docs/en_GB.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+This is a pretty simple module, designed to make it easier to send reptitive messages, this is especially useful for (co-)admins.
+
+## Chat-Messages
+
+These are messages to be inserted into the alliance chat. They won't be automatically sent, alllowing you to edit them beforehand.
+
+![chat](./chat.png)
```
Comment (Suchty112) [inline src/modules/messageTemplates/docs/en_GB.md:7]: The picture is still missing. That's why it fails when building

File: src/modules/messageTemplates/docs/en_GB.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Templates
+
+These are templated for private messages, these support the variables. 
+
+![messages](./messages.png)
```
Comment (Suchty112) [inline src/modules/messageTemplates/docs/en_GB.md:13]: same here

**Review by jxn-30 — CHANGES_REQUESTED** [review]
files `src/modules/messageTemplates/docs/assets/en_GB/centre.png` and `src/modules/messageTemplates/docs/assets/en_GB/scale.png` doesn't really make sense.
As @Suchty112 already mentioned, the `chat.png` and `messages.png` are missing instead in that folder.

**Comment by scottyboii** [comment]
Fixed those! Had uploaded extendedMap assets there by mistake instead of messageTemplates!

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**